crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -21,10 +21,8 @@ struct Case {
 
 const TOMLLIB_312_URL: &str = "https://raw.githubusercontent.com/python/cpython/8e8a4baf652f6e1cee7acde9d78c4b6154539748/Lib/tomllib";
 
-// The "unresolved import" is because we don't understand `*` imports yet.
```

Comment:
I think this comment is still accurate: this still is the reason why we're emitting the bogus `Module 'collections.abc' has no member 'Iterable'` diagnostic

Reply:
Yeah, I found it more confusing than illuminating because it talks about "unresolved import" when that wording isn't used in any error mesage, but I can re-add it with different wording (and maybe immediately above the relevant error.)

Reply:
yeah, the comment got outdated when I changed the error message. But it's still relevant.

The reason why it's not immediately above the relevant error is it's really nice to be able to copy and paste the actual diagnostics from CI straight into the file when the assertion fails :P